### ISSUE TITLE
AEM Assets Library - Filtering Issues

### ISSUE THREAD

**Issue by colinvlasak** [aemsites/stericycle-shared#626]
If I search for certain documents in the AEM Assets Library the results are not filtered properly.

I did a search for this assets. With all standard asset types showing. It showed no results.

![image](https://github.com/user-attachments/assets/678e7eee-b532-4690-a4db-d65ff5a9dcec)

I removed the image filter types (JPG, PNG, etc). 

![image](https://github.com/user-attachments/assets/8f69f28a-a5a6-43ce-bf86-434395418244)

Then it showed the correct results. Havin multiple asset types shouldn't limit the results. Taking them away should limit the results.

**Comment by jindaliiita** [comment]
@sdmcraft Is this an issue on the asset selector side ?

**Comment by MarquiseRosier** [comment]
Yeah @colinvlasak I think this probably involves asset approval flow! The assets probably weren't showing up because they weren't approved!

**Comment by jindaliiita** [comment]
@colinvlasak Meanwhile, as per [this comment](https://github.com/aemsites/stericycle-shared/issues/626#issuecomment-2592962027), I am assigning it to you.

**Comment by colinvlasak** [comment]
@MarquiseRosier @jindaliiita this is a separate issue from the approval problem. PDFs don't show up when all the filters are on. You need to remove all the image filters for them to appear.

This can be fixed post launch.

**Comment by MarquiseRosier** [comment]
Hey @sdmcraft this is a validated bug; PDFs won't show up even if you type the name correctly when all the fliters are on. Once you remove all the filters, the PDFS will suddenly appear? Something's amiss! Let me know if you need me to make a bug fix request :)
